### PR TITLE
Docs update before ph pt.1

### PR DESCRIPTION
**Pull request by fabiomaienschein**
Re-structured the menu, updated copy around suggest and discover to generate, updated screenshots regarding these topics.

### CODE REVIEW

File: generate-more-test-cases.mdx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  />
+</Frame>
+
+For each test, the AI agent has auto-generated test steps, validated them and where the validation succeeded, it turned the test **ON** - [into active mode](/execute-test-cases).
+
+Help the AI agent when it couldn't quite nail the auto-generation - **yellow alert** highlights a failed step generation. See how in the [edit test case](/edit-test-case) section.
+
+<Frame caption="New generated tests, 08/2024">
+  <img src="/images/prompting/suggested-tests.png" alt="New generated tests" />
+</Frame>
```
Comment (sker65) [inline generate-more-test-cases.mdx:37]: old screenshot with groups

File: test-case-creation-strategy.mdx [hunk 2]
```diff
@@ -6,17 +6,17 @@ icon: "lightbulb"
...
 - use `run locally` and `Playwright Codegen`
 - work on multiple test cases in parallel
 
-## Start with the AI auto-discovery
+## Start with the AI auto-generation
 
-Always use the `Discover` button on the overview page. It gives you a head start with up to 3 AI auto-discovered tests.
+Our AI auto-generation gives you a head start with up to 3 AI auto-generated tests.
 
 <Frame caption="Have AI discover first 3 test cases from overiew page, 07/2024">
```
Comment (sker65) [inline test-case-creation-strategy.mdx:21]: outdated screenshot "discover"

File: test-case-creation-strategy.mdx [hunk 3]
```diff
@@ -25,7 +25,7 @@ Always use the `Discover` button on the overview page. It gives you a head start
   />
 </Frame>
 
-You can have the AI agent auto-discover more test cases. Go into the test case and hover over the `suggest more` icon - a button will appear. Click on it. Find more on how `suggest more` works [here](/new-test-case).
+You can have the AI agent generate more test cases. Go into the test case and hover over the `generate more` icon - a button will appear. Click on it. Find more on how `generate more` works [here](/generate-more-test-cases).
 
 The AI agent will discover up to 3 new tests based on the one you started it from. The original test will be added as **dependency**. We will auto-generate steps for every discovered test case and validate them.
```
Comment (sker65) [inline test-case-creation-strategy.mdx:30]: again discover

**Review by sker65 — COMMENTED** [review]
in the test case creation strategy there are mobile screensize screenshots? Intentional?

File: first-steps.mdx [hunk 4]
```diff
@@ -75,62 +77,49 @@ Here comes the really cool part. You can let the AI agent browse through your si
   />
 </Frame>
 
-When it's done, the AI agent will let you know. You will find your tests on the `test cases` page.
+## 6. Check your site against the generated test suite
+
+We will [execute your generated test cases](/execute-test-cases.mdx) by running test reports. These will ensure that they successfully pass when executed on your site.
```
[the remark this answers is not in the document]
Comment (fabiomaienschein) [reply first-steps.mdx:82]: I like that wording

[the remark this answers is not in the document]
Comment (kosta) [reply first-steps.mdx:82]: What about removing `generated` here? We're also executing the ones that we imported from a recording :)

**Review by ma-zah — APPROVED** [review]

**Pull request merged**